Utilities/DataSampling/src/Dispatcher.cxx

```diff
@@ -77,6 +78,42 @@ void Dispatcher::init(InitContext& ctx)
   mDeviceID.runtimeInit(spec.id.substr(0, DataSamplingHeader::deviceIDTypeSize).c_str());
 }
 
+header::Stack extractAdditionalHeaders(const char* inputHeaderStack)
+{
+  std::array<header::BaseHeader const*, 5> headers;
```

Comment:
@ktf here, you used `5`, but the code below supports up to `8`. A typo?

Reply:
Yes. It used to be 5 and I then changed it to 8. Will fix this.